### PR TITLE
Fixed demos spacing.

### LINKED ISSUE #2509: Demo spacing
## Checkbox

image

image

image

## Select

image

## List
Lost spacing between items
image

Strange padding in list
image

### PR DESCRIPTION
**Pull request by Sadnessa**
## Description
Closes #2509.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/page-configs/ui-elements/list/examples/Clickable.vue [hunk 1]
```diff
@@ -49,3 +50,13 @@ export default {
...
+
+<style lang="scss">
+  #clickable-list-example {
+    padding: 0;
+
+    .list__item:not(:last-child) {
+      margin-bottom: 20px;
+    }
+  }
+</style>
```
Comment (rustem-nasyrov) [inline packages/docs/src/page-configs/ui-elements/list/examples/Clickable.vue:62]: Let's rewrite it (in other demos too) like this:
```suggestion
<style>
.list-item + .list-item {
  margin-top: 20px;
}
</style>
```

File: packages/ui/src/components/va-list/VaList.vue [hunk 2]
```diff
@@ -30,7 +30,6 @@ export default defineComponent({
   @import "variables";
 
   .va-list {
-    padding: var(--va-list-padding);
```
Comment (rustem-nasyrov) [inline packages/ui/src/components/va-list/VaList.vue:33]: This change might break styles for end users.

**Review by rustem-nasyrov — CHANGES_REQUESTED** [review]
Awesome! Thank you for your contribution, but there are still things to do.

File: packages/ui/src/components/va-list/VaList.vue (hunk 2 above)
Comment (m0ksem) [reply packages/ui/src/components/va-list/VaList.vue:33]: @rustem-nasyrov, should we care about it, if we are going to release 1.5.0?

Comment (rustem-nasyrov) [reply packages/ui/src/components/va-list/VaList.vue:33]: @m0ksem, I think for now we don't, but I'm wondering is users will notice it or not. :)

**Review by m0ksem — APPROVED** [review]

**Review by rustem-nasyrov — APPROVED** [review]

**Pull request merged**